app/jobs/modsulator_job.rb

```diff
@@ -299,7 +299,15 @@ def in_accessioning(dor_object)
   # @param  [Dor::Item]   dor_object    DOR object to check
   # @return [Boolean]     true if the object's status allows us to update the descMetadata datastream, false otherwise
   def status_ok(dor_object)
-    status = dor_object.status_info[:status_code]
+    status = get_status(dor_object) # dor_object.status_info[:status_code]
     [1, 6, 7, 8, 9].include?(status)
   end
+
+  # Returns the status_info for a DOR object from the StatusService
+  #
+  # @param  [Dor::Item]   dor_object    DOR object to check
+  # @return [Integer]     value cooresponding to the status info list
+  def get_status(dor_object)
```

Comment:
can we call this `status` or `status_info`?  the current name violates https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/AccessorMethodName

Reply:
Renamed to `status`